### DISCUSSION TITLE
Argon Pro Needs Transparency

### DISCUSSION THREAD

**Discussion by ebolite** [ppy/osu#24226] [Ideas]
First let me say that I really love argon pro. The contrast and visual clarity of the skin in most areas is almost enough for me to switch to it from my current playskin, but there is one major issue that is stopping me and probably a lot of other players from switching.

![image](https://github.com/ppy/osu/assets/22662047/36d44495-c860-4611-9299-59bd9426b446)
(From top to bottom: Argon Pro, my playskin (-GN's skin, boom), and a skin with similar hitcircles (idke 1.2))

Compared to other "pro" skins, Argon Pro is one of the only skins without transparent slider bodies and hitcircles. This is the only thing stopping me from switching, to be honest. This is a major issue for reading, since stacked objects are much more difficult to distinguish.

Changing the slider bodies and the centers of the hitcircles to be more transparent would be an immediate massive improvement to the skin's readability without compromising the skin's great visual style.
![image](https://github.com/ppy/osu/assets/22662047/cbf4208e-1293-46b1-9c27-a4c85e81880f)

**Comment by Fletscherrrr** [comment]
Honestly the hitcircles are too big to begin with. 
The hitcircleoverlay is also kinda missing a dropshadow which would also greatly improve readability for stacked bursts.
![comparison](https://github.com/ppy/osu/assets/115947237/2735c6e8-785e-483c-85ae-c2f783cb637e)
Just as a reference for what I'm talking about. It's not like it's gonna make you a reading god but it's a nice quality of life thing honestly.

**Comment by vatei** [comment]
Bumping this, I tried Argon Pro many times but it's super hard to read, which is unfortunate given how pretty everything feels.
Probably needs some tweaking as suggested above

**Comment by Siiiuuuuu** [comment]
bumping this again, especially bad on maps like these https://osu.ppy.sh/beatmapsets/669232#osu/1415526

**Comment by peppy** [comment]
Maybe resolved via https://github.com/ppy/osu/pull/33044.

**Reply by Corne2Plum3** [reply]
Could we have transparency on hit circles and slider heads as well (except on their borders), to make stacks easier to see?
